ci/utils.yml

```diff
@@ -28,7 +28,7 @@ outdated:
   retry: 2
   script:
     - yarn install --cache-folder .yarn --prefer-offline
-    - yarn workspace @trezor/suite test-health
+    - yarn workspace @trezor/urls test:unit
```

Comment:
what about `@trezor/suite test-health` we don't want to test it anymore?

Reply:
The only thing `test-health` did was to check that the urls are alive, which is exactly what unit test in `@trezor/urls` is doing currently. So I removed it.

Reply:
Personally I would rename it to "test:integration". With name "test:unit" this would be automatically picked up in unit testing phase right? Which would make it prone to random failures due to random internet glitches

Reply:
You're :100: right.
https://github.com/trezor/trezor-suite/pull/5487/commits/657d261d337b107f0a94b720bc42b8db311fd3ce
https://github.com/trezor/trezor-suite/pull/5487/commits/26532596f79d94db9d1e5f06a7df79666c7825fc